Automattic/wp-calypso
# Condensed Features Experiment

#### Proposed Changes

* Implements an A/B test on the signup plans step.
* The changes here try out a different visual treatment of plan features.
* Read pbxNRc-1WS for more information.

**Annual**

<img width="1496" alt="Screenshot 2022-09-01 at 6 03 45 PM" src="https://user-images.githubusercontent.com/1269602/187933762-db722dc7-07b4-483a-9ac9-9616a20fc1f7.png">


**Monthly**

<img width="1555" alt="Screenshot 2022-09-01 at 6 03 54 PM" src="https://user-images.githubusercontent.com/1269602/187933797-8e50a4e3-3bd8-4bb4-b966-b916e859fe78.png">



#### Testing Instructions


 
* Assign yourself to treatment of the experiment `calypso_signup_plans_step_optimize_202208_v1` in Abacus.
* Signup for a new account and verify that the plans step UI matches the one shown in screenshots above.
* Verify that mobile view is unchanged - this experiment is only for desktop, so mobile view should have no UI changes.
* Verify that non-EN is unaffected
* Verify that control version in unaffacted.
* Verify that you can continue to complete signup and create a site / complete a purchase.
